### ISSUE TITLE
Fill in pointer type on known types

### ISSUE THREAD

**Issue by thatcosmonaut** [flibitijibibo/SDL3-CS#9]
For example, `SDL_GetDesktopDisplayMode` can return `SDL_DisplayMode*` instead of IntPtr.

**Comment by Dzhake** [comment]
Sorry, I'm not familiar with c++ to c# wrappers..
Is it possible to somehow use those functions (f.e. `SDL_GetDesktopDisplayMode`) already, or I need to wait until this issue is solved?

**Comment by thatcosmonaut** [comment]
Yeah everything works fine because IntPtr can be casted to any pointer type. You just temporarily lose the type safety.
